target/linux/ath79/image/mikrotik.mk

```diff
@@ -1,5 +1,14 @@
 include ./common-mikrotik.mk
 
+define Device/mikrotik_routerboard-952ui-5a2nd
```

Comment:
This name and the DTS name don't match the string in DEVICE_MODEL and the compatible in DTS.